test/e2e/networkpolicy_test.go

```diff
@@ -215,6 +215,14 @@ func (data *TestData) setupDifferentNamedPorts(t *testing.T) (checkFn func(), cl
 		preCheckFunc(server0IPs.ipv6.String(), server1IPs.ipv6.String())
 	}
 
+	if testOptions.providerName == "kind" {
+		// Due to netdev datapath bug, sometimes datapath flows are not flushed after new openflows that change the
+		// actions are installed, causing client1 can still connect servers after creating a policy that disallows it.
```

Comment:
causing client1 to still be able to connect to the servers

Reply:
done, thanks